pkg/controller/manager/manager_controller.go

```diff
@@ -305,7 +305,10 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 				return reconcile.Result{}, err
 			}
 		}
-	} else if certOperatorManaged {
+	} else if !certOperatorManaged {
+		r.status.SetDegraded(fmt.Sprintf("self provided secret %s/%s is not supported when certificate management is not supported", render.ManagerNamespace, render.ManagerTLSSecretName), "")
```

Comment:
I think this wording is a bit off. I think maybe it should be 
> user provided secret %s/%s is not supported when certificate management is enabled

Maybe changing it to something like this? WDYT?
```
err := fmt.Errorf("user provided secret %s/%s is not supported when certificate management is enabled", render.ManagerNamespace, render.ManagerTLSSecretName)
r.status.SetDegraded("Invalid certificate configuration", errMsg)
return reconcile.Result{}, err
```